packages/framework/src/Resources/config/services/cron.yml

```diff
@@ -6,45 +6,45 @@ services:
 
     Shopsys\FrameworkBundle\Model\Product\Search\Export\ProductSearchExportCronModule:
         tags:
-            - { name: shopsys.cron, hours: '*', minutes: '*' }
+            - { name: shopsys.cron, hours: '*', minutes: '*', readableName: 'Export products to Elasticsearch' }
```

Comment:
Cannot we set the `readableName` in specific `CronModule`? I do not think it should be defined here. Also this name cannot be translatable.

Reply:
I talked with @henzigo, and as the administration cannot be in multliple languages, I'm ok with setting the name only in the config file. As the path of least resistance.